playwright/src/fixtures/MHmodal.ts

```diff
@@ -51,6 +79,55 @@ class MHHighlights {
     // Click context menu
     await this.MHContextMenu.nth(n).click()
   }
+
+  async changeColor(color: string) {
+    // Change highlight color from MH modal
+    if (color === 'blue') {
+      return this.blue.click()
+    } else if (color === 'green') {
+      return this.green.click()
+    } else if (color === 'pink') {
+      return this.pink.click()
+    } else if (color === 'purple') {
+      return this.purple.click()
+    } else if (color === 'yellow') {
+      return this.yellow.click()
+    }
+  }
+
+  async addNote(note: string) {
+    // Add note to a highlight
+    await this.addNoteMenu.click()
+    await this.page.locator('[aria-label="Enter note\\\'s text"]').type(note)
+  }
+
+  async editNote(note: string) {
+    // Edit existing note of a highlight
+    await this.editNoteMenu.click()
+    await this.page.locator('[aria-label="Enter note\\\'s text"]').type(note)
+  }
+
+
+  async clickCancel() {
+    // Click Cancel on note textbox
+   await this.cancelNote.click();
+  }
+
+  async clickSave() {
+    // Click Save on note textbox
+   await this.savelNote.click();
+  }
+
+  async clickDeleteHighlight(confirm: string) {
```

Comment:
@gregfitch is there a way in Typescript to pass the values that can be accepted for a parameter?
In `clickDeleteHighlight(confirm: string)` I want the test to send values either `delete` or `confirm`. 
Can I add these 2 vaules in the function somehow?